### PR TITLE
feat(run_server): add **kwargs App.run_server()

### PR DESCRIPTION
**Pull request by psavery**
This is forwarded to wslink.

### CODE REVIEW

File: python/src/pywebvue/core.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def protocol_call(self, method, *args, **kwargs):
     # Start server
     # -------------------------------------------------------------------------
 
-    def run_server(self, port=None, thread=False):
+    def run_server(self, port=None, thread=False, exec_mode="main"):
```
Comment (jourdain) [inline python/src/pywebvue/core.py:353]: can you add **kwargs here too and forward it to wslink?

Comment (jourdain) [reply python/src/pywebvue/core.py:353]: I would even remote `exec_mode` and just let `**kwargs` glob all of the extra ones.

Comment (psavery) [reply python/src/pywebvue/core.py:353]: Yes I can!

**Comment by jourdain** [comment]
LGTM

**Pull request merged**